blog/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+#create your models
+from django.contrib.auth.models import User
+
+
+STATUS = (
+    (0,"Draft"),
+    (1,"Publish")
+)
+
+class Post(models.Model):
+    title = models.CharField(max_length=200, unique=True)
+    slug = models.SlugField(max_length=200, unique=True)
+    author = models.ForeignKey(User, on_delete= models.CASCADE,related_name='blog_posts')
```

Comment:
You can write this as:
`author = models.ForeignKey('User', on_delete=models.CASCADE, related_name='blog_posts'`

Also read PEP 8 --> 
Equal signs in parameters must not have spaces around them